diff --git a/node_modules/@types/jsdom/ts3.3/index.d.ts b/node_modules/@types/jsdom/ts3.3/index.d.ts
index 110e4a7..c32f8e9 100644
--- a/node_modules/@types/jsdom/ts3.3/index.d.ts
+++ b/node_modules/@types/jsdom/ts3.3/index.d.ts
@@ -1,7 +1,7 @@
 /// <reference types="node" />
 
 import { EventEmitter } from 'events';
-import { ElementLocation } from 'parse5';
+import { MarkupData } from 'parse5';
 import { Context } from 'vm';
 import * as tough from 'tough-cookie';
 
@@ -9,6 +9,8 @@ declare module 'jsdom' {
     const toughCookie: typeof tough;
     class CookieJar extends tough.CookieJar {}
 
+    type ElementLocation = MarkupData.ElementLocation;
+
     class JSDOM {
         constructor(html?: string | Buffer | BinaryData, options?: ConstructorOptions);
 
