diff --git a/node_modules/@types/react-instantsearch-dom/index.d.ts b/node_modules/@types/react-instantsearch-dom/index.d.ts
index de2a9ab..3df0c37 100644
--- a/node_modules/@types/react-instantsearch-dom/index.d.ts
+++ b/node_modules/@types/react-instantsearch-dom/index.d.ts
@@ -107,7 +107,7 @@ export interface SearchBoxProps extends CommonWidgetProps {
 
   onSubmit?: (event: React.SyntheticEvent<HTMLFormElement>) => any;
   onReset?: (event: React.SyntheticEvent<HTMLFormElement>) => any;
-  onChange?: (event: React.SyntheticEvent<HTMLInputElement>) => any;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => any;
 }
 /**
  * The SearchBox component displays a search box that lets the user search for a specific query.
