patch-package
--- a/node_modules/algolia-html-extractor/lib/algoliaHtmlExtractor.js
+++ b/node_modules/algolia-html-extractor/lib/algoliaHtmlExtractor.js
@@ -48,7 +48,7 @@ module.exports = class AlgoliaHTMLExtractor {
         // Which level heading is it?
         currentLvl = parseInt(this.extractTagName(node).replace(/^h/, ''), 10) - 1;
         // Update this level, and set all the following ones to nil
-        currentHierarchy[`lvl${currentLvl}`] = this.extractText(node);
+        currentHierarchy[`lvl${currentLvl}`] = node;
 
         for (let i = currentLvl + 1; i < 6; i += 1) {
           currentHierarchy[`lvl${i}`] = null;
@@ -132,7 +132,7 @@ module.exports = class AlgoliaHTMLExtractor {
   // <h1 name="anchor">Foo</h1> => anchor
   // <h1 id="anchor">Foo</h1> => anchor
   // <h1><a name="anchor">Foo</a></h1> => anchor
-  extractAnchor(node) {
+  extractAnchor(node, _isParent = false) {
     const anchor = node.getAttribute('name') || node.getAttribute('id') || null;
 
     if (anchor) {
@@ -146,6 +146,9 @@ module.exports = class AlgoliaHTMLExtractor {
       return this.extractAnchor(subelement);
     }
 
+    if (!_isParent) {
+      return this.extractAnchor(node.parentElement, true);
+    }
 
     return null;
   }
