patch-package
--- a/node_modules/patch-package/dist/makePatch.js
+++ b/node_modules/patch-package/dist/makePatch.js
@@ -58,7 +58,7 @@ function makePatch(packageName, appPath, packageManager, includePaths, excludePa
         if (packageManager === "yarn") {
             console.info(chalk_1.default.grey("•"), "Installing " + packageDetails.name + "@" + packageVersion + " with yarn");
-            spawnSafe_1.spawnSafeSync("yarn", ["install", "--ignore-engines"], {
+            spawnSafe_1.spawnSafeSync("yarn", ["install", "--ignore-engines", "--ignore-scripts"], {
                 cwd: tmpRepoNpmRoot,
             });
         }
         else {
